packages/deployer/test/fixture-projects/sample-project/test/helpers/initializer.js

```diff
@@ -1,13 +1,13 @@
 const { getProxyAddress } = require('@synthetixio/deployer/utils/deployments');
 
-module.exports = async function initializer(deploymentInfo) {
+module.exports = async function initializer(deploymentInfo, _hre = hre) {
```

Comment:
Yes, because on the deployer tests we use fixture projects, meaning that each fixture project has its own folder and on each test we have to re-load the HRE object on the given fixture project folder (we're doing that [here](https://github.com/Synthetixio/synthetix-v3/blob/refactor-fixtures-initializers/packages/deployer/test/helpers.js#L17:L17)). I used [this boilerplate](https://github.com/nomiclabs/hardhat-ts-plugin-boilerplate/) as reference.

So, because we are loading the HRE for each fixture project and not using `hardhat  test` to run them, we do not have the global object injected at the beginning, and also, after loading it for the first time, everytime we re-load another environment the global reference to `hre` changes because we reset it. So, maybe we are keeping a reference to an old one.

Reply:
Ok cool, makes sense!